src/bun.js/bindings/bindings.zig

```diff
@@ -5872,3 +5927,30 @@ pub const ScriptExecutionStatus = enum(i32) {
     suspended = 1,
     stopped = 2,
 };
+
+pub const PropertyAttributes = c_uint;
+
+// ECMA 262-3 8.6.1
+// Property attributes
+pub const PropertyAttribute = struct {
+    pub const ReadOnly: PropertyAttributes = 1 << 1; // property can be only read, not written
+    pub const DontEnum: PropertyAttributes = 1 << 2; // property doesn't appear in (for .. in ..)
+    pub const DontDelete: PropertyAttributes = 1 << 3; // property can't be deleted
+    pub const Accessor: PropertyAttributes = 1 << 4; // property is a getter/setter
+    pub const CustomAccessor: PropertyAttributes = 1 << 5;
+    pub const CustomValue: PropertyAttributes = 1 << 6;
+};
```

Comment:
This would be a good candidate to use Zig's [packed struct](https://devlog.hexops.com/2022/packed-structs-in-zig/#zig-packed-structs).

Reply:
I though about it, but wanted to mimic the C++ definition as close as possible.
Anyways I'm gonna remove this part